summaries/sprint6.md

```diff
@@ -0,0 +1,22 @@
+# Week 6 sprint retrospective
+
+## Scrum master (JAAKIK)
+
+## Bettens, Louis
+
```

Comment:
```suggestion
 this week  i worked on implementing support for adding already existing photos from the user's device to the database. I was able to accomplish the task, but could not merge in time because of the Firebase situation.
```